crates/iroha_primitives/src/addr.rs

```diff
@@ -445,6 +445,15 @@ impl SocketAddr {
         }
     }
 
+    /// Changes port value
+    pub fn set_port(&mut self, new_port: u16) {
```

Comment:
I'd prefer to have `SocketAddr` be immutable

Reply:
Better to create a new instance with the same port.